### PR TITLE
Update GraphQL example with latest imports

### PR DESCRIPTION
**Pull request by Eprince-hub**
Update the GraphQl example reop to adopt some changes in `@apollo/experimental-nextjs-app-support` that changed how things are imported

### CODE REVIEW

File: package.json
Comment (Eprince-hub) [inline package.json:1]: Update all packages to latest

File: app/animals/dashboard/AnimalsForm.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 'use client';
 
-import { gql, useMutation } from '@apollo/client';
-import { useSuspenseQuery } from '@apollo/experimental-nextjs-app-support/ssr';
+import { gql, useMutation, useSuspenseQuery } from '@apollo/client';
```
Comment (Eprince-hub) [inline app/animals/dashboard/AnimalsForm.tsx:3]: `useSuspenseQuery` can now be imported directly from `@apollo/client`

File: app/ApolloClientProvider.tsx [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
 function makeClient() {
   const httpLink = new HttpLink({
     uri: `${process.env.NEXT_PUBLIC_BASE_URL}/api/graphql`,
     fetchOptions: { cache: 'no-store' },
   });
 
-  return new NextSSRApolloClient({
-    cache: new NextSSRInMemoryCache(),
+  return new ApolloClient({
+    cache: new InMemoryCache(),
```
Comment (Eprince-hub) [inline app/ApolloClientProvider.tsx:18]: Imports name changes

File: app/ApolloClientProvider.tsx [hunk 3]
```diff
@@ -2,20 +2,20 @@
...
 import { ApolloLink, HttpLink } from '@apollo/client';
 import {
+  ApolloClient,
   ApolloNextAppProvider,
-  NextSSRApolloClient,
-  NextSSRInMemoryCache,
+  InMemoryCache,
   SSRMultipartLink,
-} from '@apollo/experimental-nextjs-app-support/ssr';
+} from '@apollo/experimental-nextjs-app-support';
```
Comment (Eprince-hub) [inline app/ApolloClientProvider.tsx:9]: Imports are now directly on the `@apollo/experimental-nextjs-app-support` and no need for `/ssr` anymore

File: eslint.config.js [hunk 4]
```diff
@@ -1 +1,9 @@
-export { default } from 'eslint-config-upleveled';
+import upleveled from 'eslint-config-upleveled';
+
+const config = upleveled.filter(({ name }) => {
+  // Disable UpLeveled database auth config, since there
+  // is no real sessionToken in this project
+  return name !== 'upleveled:database-auth';
+});
+
+export default config;
```
Comment (Eprince-hub) [inline eslint.config.js:9]: Real `sessionToken` is not created for GraphQL lecture

**Review by karlhorky — APPROVED** [review]
seems like only import path and naming changes, LGTM

**Pull request merged**